posts/google-summer-of-code-fellows/gsoc-2024-fellows.qmd

```diff
@@ -0,0 +1,294 @@
+Initial documentations: ![](./image.png)
+New documentations using DocumenterVitepress: ![](./img2.png)
+
+
+
+**2. Loading the PostgreSQL Database**
+
+The main database we worked on/built analysis was the freely available OMOPCDM Database. However, I was provided with some more extra synthetic data from my mentor for further testing of the functionalities. Being a very large database we had to strategically download it further, my mentor helped me in setting up the Postgres on my local machine. Once, the database was set up proper testing was performed on it to check if things were as expected. With this, we were done with the database setup as well and could finally dive into the actual code logic for the Pathways synthesis.
+
+**3. Testing and Development setup on my local computer**
+ 
+To get a proper environment for functionality creation and concurrent testing we required a proper testing setup so that we could test the new functions made at the same time. This was done using [Revise.jl](https://timholy.github.io/Revise.jl/stable/). My mentor helped me set it up, added Revise.jl to the global Julia environment and made a Julia script by the name “startup.jl” and added it to “/home/jay-sanjay/.julia/config/” path in my local computer.
```

Comment:
Could you go into a little bit more depth about how each of the packages were useful for to you to some extent? Especially Revise.jl, PkgTemplates.jl, PackageCompatUI.jl -- the others you could just mention a sentence or a few words.

Reply:
I added it, do let me know if I need to add more !